packages/react-components/react-persona/Spec.md

```diff
@@ -1,63 +1,279 @@
+| :-------------------------------------------------------------------------------------------: | :-----------------------------------------------------------------------------------: |
+|                             ![](https://i.imgur.com/TuUXN7t.png)                              |                         ![](https://i.imgur.com/BbEhlc3.png)                          |
+|                             ![](https://i.imgur.com/pXEFFmu.png)                              |                         ![](https://i.imgur.com/hbms9Bp.png)                          |
+
+_Icon vs PersonaCoin_: PersonaCoin allows you to have icon only when there's no image available or it's the unknown Persona coin variant. In v9's Persona, you have the option of showcasing an icon through the Avatar or showcasing only the icon.
 
-- _Link to Open UI research_
-- _Link to comparison of v7 and v0_
-- _Link to GitHub epic issue for the converged component_
+- Note: The icon only shows up if there's no presence and size is `tiny`, `size8`, or `size16`.
+
+_Image vs PersonaCoin_: PersonaCoin allows you to have an image only within the Avatar. This can be avoided by rendering a custom coin, but there's still all the dom and styling overhead that comes with PersonaCoin. v9's Persona allows you use a single image with no extra dom.
+
+_PresenceBadge vs PersonaCoin_: PersonaCoin allows you to have only a PresenceBadge when the size of the coin is `tiny`, `size8`, or `size16`. In v9's Persona you are able to render the PresenceBadge alone.
+
+### Text lines
+
+Other than styling and naming, the text lines in Persona remain the same.
 
 ## Sample Code
 
-_Provide some representative example code that uses the proposed API for the component_
+Persona with Avatar:
+
+```jsx
+<Persona primaryText="Kevin Sturgis" secondaryText="Software Engineer" avatar={{ name: 'Kevin Sturgis' }} />
+```
+
+Persona with Avatar + PresenceBadge:
+
+```jsx
+<Persona
+  primaryText="Kevin Sturgis"
+  secondaryText="Software Engineer"
+  tertiaryText="Offline"
+  avatar={{ name: 'Kevin Sturgis' }}
+  badge={{ status: 'offline', outOfOffice: true }}
+/>
+```
+
+Persona with icon:
+
+```jsx
+<Persona primaryText="Person Call Icon" icon={<PersonCallRegular />} />
+```
+
+Persona with PresenceBadge:
+
+```jsx
+<Persona primaryText="Kevin Sturgis" badge={{ status: 'offline', outOfOffice: true }} />
+```
+
+Persona with image:
+
+```jsx
+<Persona
+  primaryText="Katri Athokas"
+  secondaryText="Software Engineer"
+  tertiaryText="CXE"
+  quaternaryText="Offline"
+  image={{ src: 'https://fabricweb.azureedge.net/fabric-website/assets/images/avatar/KatriAthokas.jpg' }}
+/>
+```
 
 ## Variants
 
-_Describe visual or functional variants of this control, if applicable. For example, a slider could have a 2D variant._
+There are three alignment variants:
+
+- start: Coin on the left and text on the right.
+- center: Coin on top and text on the bottom.
+- end: Coin on the right and text on the left.
```

Comment:
I suggest to use "media slot" or "media" as for me "coin" means a special shape and is more opinionated about its content.